app/models/benefit_coverage_period.rb

```diff
@@ -163,14 +163,9 @@ def elected_plans_by_enrollment_members(hbx_enrollment_members, coverage_kind, t
     american_indian_members = extract_american_indian_status(hbx_enrollment, shopping_family_member_ids)
 
     if EnrollRegistry.feature_enabled?(:temporary_configuration_enable_multi_tax_household_feature)
-      subjects = hbx_enrollment.family.eligibility_determination&.subjects
-
-      csr_kind = if subjects&.where(:"eligibility_states.eligibility_item_key" => 'aptc_csr_credit').present?
```

Comment:
@saikumar9 why don't we need to do this check? Is `eligibility_item_key` always going to be `aptc_csr_credit`?

Reply:
> @saikumar9 why don't we need to do this check? Is `eligibility_item_key` always going to be `aptc_csr_credit`?

CSR eligibility of a shopping group does not just depend on Financial Assistance Determination. For example, if all the members are American Indian or Alaskan Native then they are eligible for csr_limited given they are not csr_100 determined based on Financial Assistance. The `aptc_csr_credit` eligibility_state is only created when Financial Assistance is determined.